cmd/algoh/main.go

```diff
@@ -301,7 +301,7 @@ func initTelemetry(genesis bookkeeping.Genesis, log logging.Logger, dataDirector
 				}
 
 				// If the telemetry URI is not set, periodically check SRV records for new telemetry URI
-				if log.GetTelemetryURI() == "" {
+				if telemetryConfig.Enable && log.GetTelemetryURI() == "" {
```

Comment:
There may be another bug, it looks like we're already in a `if telemetryConfig.Enable` block at this point, maybe that block should be unnested?

Reply:
Yup, that's a redundant check. I suspect algoh was already fine.